app/javascript/components/subject/CaseStatus.js

```diff
@@ -161,10 +196,15 @@ class CaseStatus extends React.Component {
               <option>Continue Monitoring in Isolation Workflow</option>
             </Form.Control>
             {this.state.confirmed === 'End Monitoring' && (
-              <p className="pt-4">The monitoree will be moved into the &quot;Closed&quot; line list, and will no longer be monitored.</p>
+              <p className="pt-4">
+                The case status for the selected record will be updated to {this.state.case_status} and moved to the closed line list in the current workflow
+              </p>
             )}
             {this.state.confirmed === 'Continue Monitoring in Isolation Workflow' && (
-              <p className="pt-4">The monitoree will be moved to the isolation workflow.</p>
+              <p className="pt-4">
+                The case status for the selected record will be updated to {this.state.case_status} and moved to the appropriate line list in the isolation
```

Comment:
Each of these messages should end with a "." and also I believe "isolation" should be capitalized here. 
This is what it looks like now:
![Screen Shot 2020-09-18 at 1 55 42 PM](https://user-images.githubusercontent.com/11698457/93632327-2731d800-f9bb-11ea-93ff-772141cf80ec.png)

Reply:
done